src/app.html

```diff
@@ -5,11 +5,11 @@
     <meta http-equiv="content-type" content="text/html; charset=utf-8" />
 
     <link rel="icon" href="%sveltekit.assets%/favicon.ico" />
-    <link rel="icon" type="image/png" sizes="32x32" href="%sveltekit.assets%/favicon-32x32.png" />
-    <link rel="icon" type="image/png" sizes="16x16" href="%sveltekit.assets%/favicon-16x16.png" />
-    <link rel="sitemap" type="application/xml" href="%sveltekit.assets%/sitemap.xml" />
-    <link rel="apple-touch-icon" sizes="180x180" href="%sveltekit.assets%/apple-touch-icon.png" />
-    <link rel="mask-icon" href="%sveltekit.assets%/safari-pinned-tab.svg" color="#da532c" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
```

Comment:
Now you removed this again %sveltekit.assets%

Reply:
in the first pr header fixes if they are removed I realized and I corrected it quickly